### ISSUE TITLE
develop.element.io fails to load

### ISSUE THREAD

**Issue by babolivier** [vector-im/element-web#21601]
### Steps to reproduce

1. Load develop.element.io

#### What did you expect?

Element loads

#### What happened instead?

![image](https://user-images.githubusercontent.com/5547783/160815505-f42e9b77-3c6b-48f6-a9a0-9564ea7d96a8.png)

The spinner has been spinning for over 10min by this point. I can't see any error in the console, nor any unfinished request. I would like to rageshake but I don't know how to do that from the console (I remembered something like `mxRageshake...` but nothing with a name looking like that can be found).

### Operating system

Arch Linux

### Browser information

Firefox 98.0.2

### URL for webapp

develop.element.io

### Application version

5eecc768a389-react-4d14128d94c8-js-26cbe02a7f2e (no idea how to fish Olm's version from the console)

### Homeserver

element.io

### Will you send logs?

No

**Comment by robintown** [comment]
I've found that this can usually be fixed by a force refresh (Shift + F5).

**Comment by babolivier** [comment]
> I've found that this can usually be fixed by a force refresh (Shift + F5).

Tried that, didn't work.

I'm starting to wonder whether this is indexeddb being borked on my work session (since the logs seem to point to Element having troubles connecting to it, from my newbie eye). It feels like this was also the reason for https://github.com/vector-im/element-web/issues/21494, and earlier I had to reset my session on Element Call because it was failing to load it.
